ios/Capacitor/Capacitor/Plugins/Haptics.swift

```diff
@@ -23,6 +23,27 @@ public class CAPHapticsPlugin: CAPPlugin {
       }
     }
   }
+ 
+  @objc public func notification(_ call: CAPPluginCall) {
+      DispatchQueue.main.async {
+          if let type = call.options["type"] as? String {
+              var notificationType = UINotificationFeedbackGenerator.FeedbackType.success
+                
+              if type == "SUCCESS" {
```

Comment:
As you are setting the value to success you can skip the success if check, if it's warning or error it will become overridden, if it's success it will remain as success